README.md

```diff
@@ -432,6 +433,79 @@ Example 3:
 
 _________________________
 
+
+### qLoRA Tuning Example
+
+This method is similar to LoRA Tuning, but the base model is a quantized model.
+Set `peft_method` to `"lora"`. You can pass any of LoraConfig, see section on [LoRA Example](#lora-tuning-example).
+In addition, you can pass [LoRA quantization config](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/tuning/config/acceleration_configs/quantized_lora_config.py#L62).
+```py
+# to use auto_gptq 4bit lora base layers
+auto_gptq: AutoGPTQLoraConfig = None
+
+# to use auto_gptq 4bit lora base layers
+bnb_qlora: BNBQLoraConfig = None
+```
+
+```py
+class AutoGPTQLoraConfig:
+
+  # auto_gptq supports various kernels, to select the kernel to use.
+  kernel: str = "triton_v2"
+
+  # allow auto_gptq to quantize a model before training commences.
+  # NOTE: currently this is not allowed.
+  from_quantized: bool = True
+
+```
```

Comment:
I would not include this as it causes more confusion. Right now we only support triton kernel and from_quantized has the note that this is currently not allowed so we should not expose it to users